### DISCUSSION TITLE
Battery low vs. disconnected state changes

### DISCUSSION THREAD

**Discussion by ptr727** [RobertD502/home-assistant-flair#83] [Q&A] [answered]
I'm alerting on state changes for battery low and disconnected. I often get a battery low state right before a disconnected state.

Is that state reported by Flair or can the integration squelch such alerts if not really battery low?

**Comment by RobertD502** [comment]
I'm assuming you are producing a "battery low" state in some manual manner as the integration is only exposing voltage?

**Comment by ptr727** [comment]
This is for state changes for `device_class: battery`, but I found a bug in my automation mixing `battery` and `connectivity`, my bad.

Let me turn this in to an enhancement request then, does Flair expose battery low state, that you could report via a battery sensor, vs. just battery voltage?

**Reply by RobertD502** [reply]
I'm confused as to what the request is. Flair does not expose a battery level, only voltage - I do not know what the voltage to battery percentage correspondence is. I do know that it is not linear.

**Reply by ptr727** [reply]
If Flair expose battery low as binary state then expose `battery` binary sensor, else close comment.

**Reply by RobertD502** [reply, marked as answer]
No battery levels/battery low states are exposed by Flair, only voltage. Sorry.
